compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -82,6 +82,25 @@ def IREEGPU_DerivedThreadConfig :
   let parameters = (ins);
 }
 
+def IREEGPU_UseGlobalLoadDma :
+    AttrDef<IREEGPU_Dialect, "UseGlobalLoadDMA", [
+      DeclareAttrInterfaceMethods<IREECodegen_LoweringConfigAttrInterface, [
+        "getStaticTilingLevelSizes",
+        "getTilingLevelSizes",
+        "hasTilingLevel",
+      ]>
+    ]> {
+  let mnemonic = "use_global_load_dma";
+  let summary = [{
+    drive lowering of an operation by using global load DMA.
+  }];
+  let description = [{
+    TODO: Add description.
```

Comment:
Would be good to address this TODO before landing.

Reply:
+1, it should be a requirement. Otherwise, please add a reason about why it is a TODO.

Reply:
apologies, those TODOs are updated.